README.md

```diff
@@ -440,20 +449,85 @@ const builder = new CFDefinitionsBuilder([
 #### Example output
 
 ```typescript
-import type { ChainModifiers, Entry, EntryFieldTypes, EntrySkeletonType, LocaleCode } from "contentful";
+import type {
+  ChainModifiers,
+  Entry,
+  EntryFieldTypes,
+  EntrySkeletonType,
+  LocaleCode,
+} from 'contentful';
 
 export interface TypeAnimalFields {
   bread?: EntryFieldTypes.Symbol;
 }
 
-export type TypeAnimalSkeleton = EntrySkeletonType<TypeAnimalFields, "animal">;
-export type TypeAnimal<Modifiers extends ChainModifiers, Locales extends LocaleCode> = Entry<TypeAnimalSkeleton, Modifiers, Locales>;
+export type TypeAnimalSkeleton = EntrySkeletonType<TypeAnimalFields, 'animal'>;
+export type TypeAnimal<Modifiers extends ChainModifiers, Locales extends LocaleCode> = Entry<
+  TypeAnimalSkeleton,
+  Modifiers,
+  Locales
+>;
 
-export function isTypeAnimal<Modifiers extends ChainModifiers, Locales extends LocaleCode>(entry: Entry<EntrySkeletonType, Modifiers, Locales>): entry is TypeAnimal<Modifiers, Locales> {
-  return entry.sys.contentType.sys.id === 'animal'
+export function isTypeAnimal<Modifiers extends ChainModifiers, Locales extends LocaleCode>(
+  entry: Entry<EntrySkeletonType, Modifiers, Locales>,
+): entry is TypeAnimal<Modifiers, Locales> {
+  return entry.sys.contentType.sys.id === 'animal';
 }
 ```
 
+## ResponseTypeRenderer
+
+Adds response types for every content type which are compatible with contentful.js v10.
+
+#### Example Usage
+
+```typescript
+import {
+  CFDefinitionsBuilder,
+  V10ContentTypeRenderer,
+  ResponseTypeRenderer,
+} from 'cf-content-types-generator';
+
+const builder = new CFDefinitionsBuilder([
+  new V10ContentTypeRenderer(),
+  new ResponseTypeRenderer(),
+]);
+```
+
+#### Example output
+
+```typescript
+import type {
+  ChainModifiers,
+  Entry,
+  EntryFieldTypes,
+  EntrySkeletonType,
+  LocaleCode,
+} from 'contentful';
+
+export interface TypeAnimalFields {
+  bread?: EntryFieldTypes.Symbol;
+}
+
+export type TypeAnimalSkeleton = EntrySkeletonType<TypeAnimalFields, 'animal'>;
+export type TypeAnimal<Modifiers extends ChainModifiers, Locales extends LocaleCode> = Entry<
+  TypeAnimalSkeleton,
+  Modifiers,
+  Locales
+>;
+
+export type TypeAnimalWithoutLinkResolutionResponse = TypeAnimal<'WITHOUT_LINK_RESOLUTION'>;
+export type TypeAnimalWithUnresolvableLinksResponse = TypeAnimal<'WITHOUT_UNRESOLVABLE_LINKS'>;
```

Comment:
```suggestion
export type TypeAnimalWithoutUnresolvableLinksResponse = TypeAnimal<'WITHOUT_UNRESOLVABLE_LINKS'>;
```

Reply:
Btw, should we also have a type with unresolvable links?

Reply:
@veu but isn't that the default?

Reply:
It’s the default setting when running contentful.js but there is no default for the type parameter `Modifiers` in the generated type here so you can’t just omit it.